modules/Total/index.js

```diff
@@ -0,0 +1,28 @@
+const getEmote = (emote, discord) => {
+  return discord ? `:${emote}:` : emote;
+};
+
+const getBold = (string, discord) => {
+  return discord ? `**${string}**` : string;
+};
+
+module.exports = {
+  name: 'Total',
+  module(jaffamod) {
+    jaffamod.registerCommand('total', (message, reply, discord) => {
+      // Determine if it's JingleJam time
+      //  (December + first 7 days of the new year to view the total raised)
+      const d = new Date();
+      if (d.getMonth() === 11 || (d.getMonth() === 0 && d.getDate() <= 7)) {
+        jaffamod.api.get('https://jinglejam.yogscast.com/api/total').catch(() => {
```

Comment:
This method was referenced in an example in the docs, I assume it fetches any valid JSON endpoint and returns the parsed data?